### PR TITLE
Added new SDK stack

### PR DESCRIPTION
**Pull request by piotrwitek**
Added new SDK stack deployment:
- new stack definition that is run with "sst:deploy:sdk:staging" or "sst:deploy:sdk:prod"
- new github actions that is run only manually from the selected branch
- sdk will not deploy automatically from now on, only manually

### CODE REVIEW

File: sst.config.ts [hunk 1]
```diff
@@ -81,6 +88,8 @@ const runDockerCompose = async () => {
 
 export const sstConfig: SSTConfig = {
   async config(_input) {
+    const { app } = _input as any
```
Comment (robercano) [inline sst.config.ts:91]: explicit disable of eslint any?

File: sst.config.ts [hunk 2]
```diff
@@ -133,11 +142,7 @@ export const sstConfig: SSTConfig = {
       throw new Error('Please specify stage or set SST_USER env variable')
     }
 
-    if (
-      _input.stage &&
-      !availableStage.includes(_input.stage) &&
-      !_input.stage.startsWith('dev-')
-    ) {
+    if (_input.stage && !availableStage.includes(_input.stage)) {
```
Comment (robercano) [inline sst.config.ts:145]: Do we want to disable dev stages to be deployed?

**Review by robercano — APPROVED** [review]
Looks good! A couple of questions but feel free to merge and we can discuss the comments when you are back

File: sst.config.ts (hunk 1 above)
Comment (piotrwitek) [reply sst.config.ts:91]: This is the best I can do

File: sst.config.ts (hunk 2 above)
Comment (piotrwitek) [reply sst.config.ts:146]: Let me know if you have reasons to need that perhaps I can propose better alternative.

Comment (robercano) [reply sst.config.ts:146]: No, looks good, just making sure it was not a bug

File: sst.config.ts (hunk 1 above)
Comment (robercano) [reply sst.config.ts]: Oh, I was just suggesting to add an eslint-disable-next-line rule to avoid linting errors/warnings

**Pull request merged**